lvresize: add vg to positional arg and example extend (#5150)
tldr-pages/tldr: pages/linux/lvresize.md

# lvresize

> Change the size of a logical volume.
> More information: <https://man7.org/linux/man-pages/man8/lvresize.8.html>.

- Change the size of a logical volume to 120GB:

`lvresize --size {{120G}} {{volume_group}}/{{logical_volume}}`

> - Extend the size of a logical volume by 120GB as well as the underlying filesystem:

Reviewer: ```suggestion
- Extend the size of a logical volume as well as the underlying filesystem by 120GB:
```

Author: just committed your suggestions the proper way. thanks!

`lvresize --size +{{120G}} --resizefs {{volume_group}}/{{logical_volume}}`

- Extend the size of a logical volume to 100% of the free physical volume space:

`lvresize --size {{100}}%FREE {{volume_group}}/{{logical_volume}}`

> - Reduce the size of a logical volume by 120GB as well as the underlying filesystem:

Reviewer: ```suggestion
- Reduce the size of a logical volume as well as the underlying filesystem by 120GB:
```

> Change the size of a logical volume.

> - Change a logical volume's size to 120GB:

Reviewer: ```suggestion
- Change the size of a logical volume to 120GB:
```

`lvresize --size {{120G}} {{volume_group}}/{{logical_volume}}`

> - Extend a logical volume's size by 120GB as well as the underlying filesystem:

Reviewer: ```suggestion
- Extend the size of a logical volume by 120GB as well as the underlying filesystem:
```

Author: thanks!

> `lvresize --size -{{120G}} --resizefs {{volume_group}}/{{logical_volume}}`

Reviewer 2: ```suggestion
`lvresize --size +{{120G}} --resizefs {{volume_group}}/{{logical_volume}}`
```

Author: i really overlooked that with the copy paste haha, thanks!

> - Extend a logical volume's size to 100% of the free physical volume space:

Reviewer: ```suggestion
- Extend the size of a logical volume to 100% of the free physical volume space:
```

`lvresize --size {{100}}%FREE {{volume_group}}/{{logical_volume}}`

> - Reduce a logical volume's size by 120GB as well as the underlying filesystem:

Reviewer: ```suggestion
- Reduce the size of a logical volume by 120GB as well as the underlying filesystem:
```
